clusterloader2/pkg/measurement/common/api_availability_measurement.go

```diff
@@ -252,7 +259,10 @@ func (a *apiAvailabilityMeasurement) gather() ([]measurement.Summary, error) {
 	}
 	summary := measurement.CreateSummary(apiAvailabilityMeasurementName, "json", content)
 	a.summaries = append(a.summaries, summary)
-	return a.summaries, nil
+	if sli := output.ClusterSummary.AvailabilityPercentage; sli < a.threshold {
+		err = fmt.Errorf("API availability SLO not fulfilled (SLI: %.2f, SLO: %.2f)", sli, a.threshold)
```

Comment:
Could you make it more readable, without so many acronyms? 
like "got: x, expected >= x"

Reply:
Done.